pkg/cmd/server/start/start_kube_controller_manager.go

```diff
@@ -100,7 +100,7 @@ func newKubeControllerManager(kubeconfigFile, saPrivateKeyFile, saRootCAFile, po
 		cmdLineArgs["cluster-signing-key-file"] = []string{""}
 	}
 	if _, ok := cmdLineArgs["experimental-cluster-signing-duration"]; !ok {
-		cmdLineArgs["experimental-cluster-signing-duration"] = []string{"0s"}
+		cmdLineArgs["experimental-cluster-signing-duration"] = []string{"720h"}
```

Comment:
I'd prefer much shorter to flush out refreshing errors.  o(12 hours)

Reply:
It doesn't work in 3.7 for client certs so we're going to do it on the node level or backport the required changes.  Also, the goal is to get to bootstrapping first, then tighten rotation, so I don't want to block on that.

Reply:
> It doesn't work in 3.7 for client certs so we're going to do it on the node level or backport the required changes. Also, the goal is to get to bootstrapping first, then tighten rotation, so I don't want to block on that.

This looks like a time bomb.  Are you saying you're going to be manually refereshing certificates or something?

Reply:
Uh so what happens after a month?